run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+docker run -it -v "$(pwd)"/dist:/dist_ext -p 8000:8000 torat
```

Comment:
Doesn't the Dockerfile also need an `EXPOSE 8000` directive for this to work?

Reply:
it doesnt need it to be declared inside the Dockerfile, only if you dont want an explicit Portforward. In this case its not yet nearly as finished as it could be, so its maybe better to not expose it yet